src/fq/fq_filter/config.vsh.yaml

```diff
@@ -0,0 +1,58 @@
+name: fq_filter
+namespace: fq
+description: Filters a FASTQ file based on record names or sequence patterns.
+keywords: [fastq, filter, sequence]
```

Comment:
I prefer to include `commands` in here when we wrap a cli util

Reply:
OH! I see ;)